index.js

```diff
@@ -111,11 +101,11 @@ app.post('/products/add', (req, res) => {
 // TODO: need a way to look up product by name or other identifier than description
 app.put('/products/updateProductDescription', (req, res) => {
   const { productDescription, id } = req.body;
-  con.query(`UPDATE Product SET productDescription = ${productDescription} WHERE ID = ${id}`, (err, status) => {
+  pool.query(`UPDATE products SET description = ${productDescription} WHERE id = ${id}`, (err, status) => {
     if (err) {
-      res.send('Err updating description: ', err);
-    } else if (status.affectedRows) {
-      res.sendStatus(200);
+      res.send(err);
+    } else if (status.rowCount) {
+      res.send(status);
     } else {
       res.status(404).send('Failed to update product');
     }
```

Comment:
The refactoring looks good to me - don't really have any comment this time around